terpnetwork/chain.json

```diff
@@ -102,7 +102,10 @@
         "consensus": {
           "type": "cometbft",
           "version": "0.37.2"
-        }
+        },
+        "next_version_name": "v4.1.0",
+        "proposal": 30,
+        "height": 3698609
```

Comment:
107 and 108 (prop and height) should be moved to the next section for v4.1.0. Then we can merge.